crypto/x509/x509_vpm.c

```diff
@@ -502,6 +502,17 @@ const char *X509_VERIFY_PARAM_get0_name(const X509_VERIFY_PARAM *param)
  */
 
 static const X509_VERIFY_PARAM default_table[] = {
+    {
```

Comment:
Are you sure this item can be added in front of the list ?

Reply:
Yes, this table is sorted by parameters name.